roles/infrastructure/tasks/vms/create.yml

```diff
@@ -26,7 +26,7 @@
     name: "nic-{{ deployment_id }}-{{ app_tag }}-{{ infrastructure_vm_deployment_id }}"
     virtual_network: "{{ infrastructure_network.state.id }}"
     subnet: "{{ subnet }}"
-    public_ip_name: "{{ infrastructure_pip.state.name }}"
+    #public_ip_name: "{{ infrastructure_pip.state.name }}"
```

Comment:
Rather than commenting the line, let's just remove it since 2.4.0 doesn't support it.